source/common/upstream/load_balancer_impl.h

```diff
@@ -694,7 +694,9 @@ class LeastRequestLoadBalancer : public EdfLoadBalancerBase {
             least_request_config.has_value() && least_request_config->has_active_request_bias()
                 ? absl::optional<Runtime::Double>(
                       {least_request_config->active_request_bias(), runtime})
-                : absl::nullopt) {
+                : absl::nullopt),
+        enable_full_scan_(
+            PROTOBUF_GET_WRAPPED_OR_DEFAULT(least_request_config.ref(), enable_full_scan, false)) {
```

Comment:
> I'm not 100% sure how to do this on the code, when removing the field from here I was having problems compiling the load_balancer_impl.* files. I would appreciate help on this.

Just keep this `enable_full_scan_` always be false if legacy API is used. Then I think this will resolve the compiling problem.

/wait